bindings/python/mainloop.pyx

```diff
@@ -0,0 +1,13 @@
+
+cdef extern from "sol-mainloop.h":
+	int sol_init()
+	int sol_run()
+
+class SolettaMainloop():
+	def __init__(self):
+		sol_init()
+
```

Comment:
Agreed.  These are just "stubs" for now while I get cython integrated with the build system.  I wouldn't consider anything here an actual API at this point.